shared/fs/row/still.js

```diff
@@ -0,0 +1,147 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import {globalColors, globalMargins, isMobile, glamorous} from '../../styles'
+import rowStyles from './styles'
+import {Badge, Box, ClickableBox, Icon, Meta, Text, Divider} from '../../common-adapters'
+import PathItemIcon from '../common/path-item-icon'
+import PathItemInfo from '../common/path-item-info'
+
+type StillProps = {
+  name: string,
+  type: Types.PathType,
+  lastModifiedTimestamp: number,
+  lastWriter: string,
+  shouldShowMenu: boolean,
+  itemStyles: Types.ItemStyles,
+  badgeCount: number,
+  tlfMeta?: Types.FavoriteMetadata,
+  resetParticipants: Array<string>,
+  isUserReset: boolean,
+  onOpen: () => void,
+  openInFileUI: () => void,
+  onAction: (event: SyntheticEvent<>) => void,
+}
+
+const HoverBox = isMobile
+  ? Box
+  : glamorous(Box)({
+      '& .fs-path-item-hover-icon': {
+        color: globalColors.white,
+      },
+      ':hover .fs-path-item-hover-icon': {
+        color: globalColors.black_40,
+      },
+      '& .fs-path-item-hover-icon:hover': {
+        color: globalColors.black_60,
+      },
+    })
+
+const RowMeta = ({badgeCount, isNew, isIgnored, needsRekey, resetParticipants}) => {
+  if (isIgnored || !(isNew || isIgnored || needsRekey || badgeCount || resetParticipants)) {
+    return <Box />
```

Comment:
why box vs null?

Reply:
I think this might be from before the refactor but I’ll try changing it to null.